Functions.Templates/Templates/DurableFunctionsHttpStart-JavaScript/metadata.json

```diff
@@ -0,0 +1,18 @@
+﻿{
+  "defaultFunctionName": "DurableFunctionsHttpStart",
+  "description": "$DurableFunctionsHttpStart_description",
+  "name": "Durable Functions HTTP starter",
+  "language": "JavaScript",
+  "category": [
+    "$temp_category_durableFunctions"
+  ],
+  "categoryStyle": "other",
+  "enabledInTryMode": false,
+  "userPrompt": [],
+  "extensions": [
+    {
+      "id": "Microsoft.Azure.WebJobs.Extensions.DurableTask",
+      "version": "1.6.2"
```

Comment:
Will these templates work with 1.6.2, or will they require 1.7.0?

Reply:
Good catch, thank you.